### DISCUSSION TITLE
Video Download Helper - How to name file

### DISCUSSION THREAD

**Discussion by BlackBeardPirate** [aclap-dev/video-downloadhelper#580] [Help and Q&A]
### Your question

Just upgraded to the new and improved Video Download Helper.
Ok, I give up.  How do I assign a name to the file prior to initiating download?
If it's there, it is not intuitive nor obvious....at least to me.
It's gotta be there.   Feel like a daaaaaah.

### Addon & CoApp version and details

Version 9.0.1.13 (stable), target google, locale en-US.
Companion App installed (v2.0.19) Recheck
/Applications/net.downloadhelper.coapp.app/Contents/MacOS/vdhcoapp
macOS Sonoma 14.4.1
Chrome Version 124.0.6367.208

**Comment by paulrouget** [comment]
Use "Download As" (under the 3 dots menu)? Does that do the trick for you:

<img width="330" alt="Screenshot 2024-05-24 at 01 01 37" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/aa36c0d4-a8fa-4cf0-ae94-b0a0d51ad24a">

**Reply by BlackBeardPirate** [reply]
Daaah again.  Those tiny 3 dots to the right of "Download" that only pubescent eyes could see.  Certainly not these wornout 75 year old eyes. :-(
But thank you.  I see that option now.

**Reply by paulrouget** [reply]
Ah! Sorry about that, in version 9.0.1.20 (coming in a couple of days) we will have an option for bigger text!
